example/docker-helper/docker_helper/helper.py

```diff
@@ -9,6 +10,12 @@
 
 class DockerHelper(object):
     def setup(self):
+        # Filter out unclosed socket ResourceWarnings. These happen due to
+        # requests' connection pooling and spam the logs unnecessarily:
+        # https://github.com/requests/requests/issues/1882
```

Comment:
I don't think there's even an obvious way to tell requests not to reuse connections. You would have to create `Session` objects for each request to achieve that afaics.